packages/lightning-lsp-common/src/resources/sfdx/tsconfig-sfdx.base.json

```diff
@@ -0,0 +1,13 @@
+{
+    "typeAcquisition": {
+        "include": [
+            "jest"
+        ]
+    },
```

Comment:
```suggestion
```

> Type Acquisition is only important for JavaScript projects. In TypeScript projects you need to include the types in your projects explicitly. However, for JavaScript projects, the TypeScript tooling will download types for your modules in the background and outside of your node_modules folder.

As per the [TypeScript docs](https://www.typescriptlang.org/tsconfig/#typeAcquisition), `typeAcquisition` isn't needed for TypeScript projects.